WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -740,6 +748,16 @@ public void onClick(DialogInterface dialog, int which) {
                 AnalyticsTracker.Stat.SITE_SETTINGS_EXPORT_SITE_ACCESSED);
     }
 
+    private void dismissProgressDialog(ProgressDialog progressDialog) {
```

Comment:
I don't like this solution much, but I don't see any better. We could  use a EventBus to make fragment transaction when the activity is not paused.